lang/en.json

```diff
@@ -80,5 +80,6 @@
   "Done": "Done",
   "Caption Settings Dialog": "Caption Settings Dialog",
   "Beginning of dialog window. Escape will cancel and close the window.": "Beginning of dialog window. Escape will cancel and close the window.",
-  "End of dialog window.": "End of dialog window."
+  "End of dialog window.": "End of dialog window.",
+  "{1} is loading.": "{1} is loading."
```

Comment:
can you regenerate `grunt check-translations`?

Reply:
Done